README.md

```diff
@@ -62,7 +62,8 @@ details of the changes between versions.
 | **Version** | **Release Date** | **Source** | **Documentation** | **Download** |
 |:-------:|:------:|:-------------:|:-------:|:------------:|
 | **master** (unstable) | -- | [source](https://github.com/Unity-Technologies/ml-agents/tree/master) |  [docs](https://github.com/Unity-Technologies/ml-agents/tree/master/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/master.zip) |
-| **0.14.1** (latest stable release) | February 26, 2020 | **[source](https://github.com/Unity-Technologies/ml-agents/tree/latest_release)** |  **[docs](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Readme.md)** | **[download](https://github.com/Unity-Technologies/ml-agents/archive/latest_release.zip)** |
+| **0.15.0** | March 18, 2020 | **[source](https://github.com/Unity-Technologies/ml-agents/tree/latest_release)** |  **[docs](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Readme.md)** | **[download](https://github.com/Unity-Technologies/ml-agents/archive/latest_release.zip)** |
+| **0.14.1**  | February 26, 2020 | [source](https://github.com/Unity-Technologies/ml-agents/tree/0.14.1) |  [docs](https://github.com/Unity-Technologies/ml-agents/tree/0.14.1/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/0.14.1) |
```

Comment:
missing .zip in the download link

Reply:
I feel like CI should have caught this

Reply:
CI only checks remote links daily, not on PRs. This was due to some links being flaky (often Unity blog posts).

If you want to try messing with regexes to scan `https://github.com` on PRs, they're here:
https://github.com/Unity-Technologies/ml-agents/blob/master/markdown-link-check.fast.json (runs on PRs)
https://github.com/Unity-Technologies/ml-agents/blob/master/markdown-link-check.full.json (runs nightly)

There's also a script to generate the table: https://github.com/Unity-Technologies/ml-agents/blob/master/utils/make_readme_table.py